src/corelibs/U2Gui/U2Gui.qrc

```diff
@@ -162,6 +162,8 @@
         <file>images/sort_ascending.png</file>
         <file>images/ssp_logo.png</file>
         <file>images/sum.png</file>
+        <file>images/sync-msa-on.png</file>
```

Comment:
These icons have too small resolution. There was a letter from a user several days ago about small icons on Windows if the system "Scale" parameter is too large - this problem has appeared because of the fact, that almost all icons are 16x16 (some icons we have are 512x512 and this problem isn't appear to them). So, could you improve the resolution of the icon and make it 512x512?

Reply:
I agree about the icon size: we should re-design all our icons. The icons in this PR are the same as all other icons. 

I would fix all icons separately and made them SVG (vector) if possible